content/en/serverless/troubleshooting/installing_the_forwarder.md

```diff
@@ -0,0 +1,354 @@
+---
```

Comment:
I don't know why this file keeps showing up in this PR.  Every branch I have has this file moving between the /troubleshooting directory and /installation

Reply:
It's now merged in the installation folder on the main branch

Reply:
Okay - hopefully merging this PR in doesn't undo that LOL

Reply:
boom fixed